{"date":"2019-11-06T11:28:37.129","level":"ERROR","category":"recordUe2Usdt.js","host":"frank-hp","pid":"54616","data":'err from recordTransfer() to insert db error,ths track is { SequelizeValidationError: notNull Violation: Eth_charge.is_queue cannot be null
    at Promise.all.then (D:\work_space\yuezhi\eos\exchange\node_modules\sequelize\lib\instance-validator.js:74:15)
    at tryCatcher (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:517:31)
    at Promise._settlePromise (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:574:18)
    at Promise._settlePromise0 (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:619:10)
    at Promise._settlePromises (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:699:18)
    at Promise._fulfill (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:643:18)
    at PromiseArray._resolve (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise_array.js:126:19)
    at PromiseArray._promiseFulfilled (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise_array.js:144:14)
    at Promise._settlePromise (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:579:26)
    at Promise._settlePromise0 (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:619:10)
    at Promise._settlePromises (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:699:18)
    at _drainQueueStep (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:138:12)
    at _drainQueue (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:131:9)
    at Async._drainQueues (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
  name: 'SequelizeValidationError',
  errors:
   [ ValidationErrorItem {
       message: 'Eth_charge.is_queue cannot be null',
       type: 'notNull Violation',
       path: 'is_queue',
       value: null,
       origin: 'CORE',
       instance: [Eth_charge],
       validatorKey: 'is_null',
       validatorName: null,
       validatorArgs: [] } ] }:'}
{"date":"2019-11-06T11:29:04.154","level":"ERROR","category":"recordUe2Usdt.js","host":"frank-hp","pid":"54616","data":'err from recordTransfer() to insert db error,ths track is { SequelizeValidationError: notNull Violation: Eth_charge.is_queue cannot be null
    at Promise.all.then (D:\work_space\yuezhi\eos\exchange\node_modules\sequelize\lib\instance-validator.js:74:15)
    at tryCatcher (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:517:31)
    at Promise._settlePromise (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:574:18)
    at Promise._settlePromise0 (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:619:10)
    at Promise._settlePromises (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:699:18)
    at Promise._fulfill (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:643:18)
    at PromiseArray._resolve (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise_array.js:126:19)
    at PromiseArray._promiseFulfilled (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise_array.js:144:14)
    at Promise._settlePromise (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:579:26)
    at Promise._settlePromise0 (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:619:10)
    at Promise._settlePromises (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\promise.js:699:18)
    at _drainQueueStep (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:138:12)
    at _drainQueue (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:131:9)
    at Async._drainQueues (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (D:\work_space\yuezhi\eos\exchange\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
  name: 'SequelizeValidationError',
  errors:
   [ ValidationErrorItem {
       message: 'Eth_charge.is_queue cannot be null',
       type: 'notNull Violation',
       path: 'is_queue',
       value: null,
       origin: 'CORE',
       instance: [Eth_charge],
       validatorKey: 'is_null',
       validatorName: null,
       validatorArgs: [] } ] }:'}
{"date":"2019-11-06T16:03:05.786","level":"ERROR","category":"recordUe2Usdt.js","host":"frank-hp","pid":"77864","data":'err from recordTransfer() to insert db error,ths track is { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (D:\work_space\yuezhi\eos\exchange\node_modules\sequelize\lib\dialects\mysql\query.js:223:16)
    at Execute.handler [as onResult] (D:\work_space\yuezhi\eos\exchange\node_modules\sequelize\lib\dialects\mysql\query.js:51:23)
    at Execute.execute (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\commands\command.js:30:14)
    at Connection.handlePacket (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:408:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:70:12)
    at PacketParser.executeStart (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:77:25)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'pog_txid must be unique',
       type: 'unique violation',
       path: 'pog_txid',
       value:
        '4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4',
       origin: 'DB',
       instance: [Eth_charge],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields:
   { pog_txid:
      '4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4' },
  parent:
   { Error: Duplicate entry '4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4' for key 'pog_txid'
       at Packet.asError (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\packets\packet.js:708:17)
       at Execute.execute (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\commands\command.js:28:26)
       at Connection.handlePacket (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:408:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:70:12)
       at PacketParser.executeStart (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:77:25)
       at Socket.emit (events.js:198:13)
       at addChunk (_stream_readable.js:288:12)
       at readableAddChunk (_stream_readable.js:269:11)
       at Socket.Readable.push (_stream_readable.js:224:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage:
      'Duplicate entry \'4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4\' for key \'pog_txid\'',
     sql:
      'INSERT INTO `Eth_charge` (`id`,`from_eth_address`,`usdt_value`,`ue_value`,`recharge_time`,`is_exchanged`,`is_queue`,`pog_blockNumber`,`pog_confirm_blockNumber`,`pog_account`,`pog_txid`,`service_charge`,`exchange_rate`,`log_info`) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?);',
     parameters:
      [ 'uBRL6O9P',
        '0xd73b2c72435276c98a1266db457fbcd0c2d2bf28',
        '15',
        '20',
        '2019-11-06 08:03:05',
        false,
        false,
        942503,
        942509,
        '11g22p33s',
        '4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4',
        '5 USDT',
        '1',
        'UE2USDT' ] },
  original:
   { Error: Duplicate entry '4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4' for key 'pog_txid'
       at Packet.asError (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\packets\packet.js:708:17)
       at Execute.execute (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\commands\command.js:28:26)
       at Connection.handlePacket (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:408:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:70:12)
       at PacketParser.executeStart (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (D:\work_space\yuezhi\eos\exchange\node_modules\mysql2\lib\connection.js:77:25)
       at Socket.emit (events.js:198:13)
       at addChunk (_stream_readable.js:288:12)
       at readableAddChunk (_stream_readable.js:269:11)
       at Socket.Readable.push (_stream_readable.js:224:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage:
      'Duplicate entry \'4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4\' for key \'pog_txid\'',
     sql:
      'INSERT INTO `Eth_charge` (`id`,`from_eth_address`,`usdt_value`,`ue_value`,`recharge_time`,`is_exchanged`,`is_queue`,`pog_blockNumber`,`pog_confirm_blockNumber`,`pog_account`,`pog_txid`,`service_charge`,`exchange_rate`,`log_info`) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?);',
     parameters:
      [ 'uBRL6O9P',
        '0xd73b2c72435276c98a1266db457fbcd0c2d2bf28',
        '15',
        '20',
        '2019-11-06 08:03:05',
        false,
        false,
        942503,
        942509,
        '11g22p33s',
        '4c8c4cea802f69fc648eef88a47264f42b8664b2c2f9166502bdc6c17d9199f4',
        '5 USDT',
        '1',
        'UE2USDT' ] },
  sql:
   'INSERT INTO `Eth_charge` (`id`,`from_eth_address`,`usdt_value`,`ue_value`,`recharge_time`,`is_exchanged`,`is_queue`,`pog_blockNumber`,`pog_confirm_blockNumber`,`pog_account`,`pog_txid`,`service_charge`,`exchange_rate`,`log_info`) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?);' }:'}
